code/model/model.py

```diff
@@ -112,13 +115,21 @@ def forward_encoder(self,seq,lengths):
             kl_loss = kl_loss + self.KL_loss(z_mu_q[i], z_var_q[i], z_mu_p[i], z_var_p[i])
         return z_sample[0],kl_loss
 
-
     def forward(self, batch):
         seq,lengths = batch
         z,KL_loss= self.forward_encoder(seq,lengths)
         recon_loss = self.decoder(batch,z)
```

Comment:
这个的forward_encoder 是不是拆分掉比较好？总会觉得跟encoder的forward看串了，建议改成forward_lattent之类的？ 这样看上去就是三段式：
```suggestion
        h= self.encoder(batch) # batch在encoder内部拆分为seq和leng
        z_sample, KL_loss = self.forward_latent(h)
        recon_loss = self.decoder(batch,z)
```

Reply:
嗯嗯，好的